src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  hidden = false;
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  checkHealth() {
+    if (this.health <= 0) {
+      Animal.alive = Animal.alive.filter((animal) => animal !== this);
+    }
+  }
```

Comment:
The 'checkHealth' method is responsible for removing the animal from the 'alive' array. This is good, but it's not clear from the method's name what it does. Consider renaming it to something more descriptive, like 'removeIfDead'.

Reply:
Fix this comment
Your function should better describe what it do